sources/xfscli/src/bin/xfs_admin.rs

```diff
@@ -0,0 +1,212 @@
+use std::process::exit;
```

Comment:
nit: probably this commit should be before the other two commits, since in this commit `query_xfs_db` and `get_mount_point` were defined.